imap-backup.gemspec

```diff
@@ -8,8 +9,9 @@ Gem::Specification.new do |gem|
   gem.authors       = ["Joe Yates"]
   gem.email         = ["joe.g.yates@gmail.com"]
   gem.homepage      = "https://github.com/joeyates/imap-backup"
-
-  gem.files         = `git ls-files`.split($OUTPUT_RECORD_SEPARATOR)
+  gem.licenses      = ["MIT"]
```

Comment:
As this is a separate matter from the listing functionality, could it go in a separate commit?